Tasks/HelmDeployV0/Tests/L0.ts

```diff
@@ -112,6 +112,27 @@ describe("HelmDeployV0 Suite", function () {
         done();
     });
 
+    it("Run successfully with Helm install (version 3) when mulitple value files are given", function(done: MochaDone) {        
+        const tp = path.join(__dirname, "TestSetup.js");
+        const tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+        process.env[shared.TestEnvVars.connectionType] = shared.ConnectionTypes.KubernetesServiceConnection;
+        process.env[shared.TestEnvVars.command] = shared.Commands.install;
+        process.env[shared.TestEnvVars.chartType] = shared.ChartTypes.Name;
+        process.env[shared.TestEnvVars.chartName] = shared.testChartName;
+        process.env[shared.TestEnvVars.valueFile] = shared.valueFile;
```

Comment:
call it `multipleValueFiles` or something on those lines maybe?

Add a test for validating single value file as well

Reply:
Ok